Migrating some more callsites

tugsbayasgalan opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #165332
* __->__ #163580
* #165582


cc @voznesenskym @penguinwu @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @wenzhe-nrv @jiayisunx @ipiszy @chenyang78 @kadeng @muchulee8 @amjames @chauhang @aakhundov @coconutruben @Lucaskabela

tugsbayasgalan committed:
Migrating some more callsites

[ghstack-poisoned]

tugsbayasgalan committed:
Update on "Migrating some more callsites"


Differential Revision: [D83009423](https://our.internmc.facebook.com/intern/diff/D83009423)

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx ipiszy chenyang78 kadeng muchulee8 amjames chauhang aakhundov coconutruben Lucaskabela

[ghstack-poisoned]

tugsbayasgalan committed:
Update on "Migrating some more callsites"


Differential Revision: [D83009423](https://our.internmc.facebook.com/intern/diff/D83009423)

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx ipiszy chenyang78 kadeng muchulee8 amjames chauhang aakhundov coconutruben Lucaskabela

[ghstack-poisoned]

tugsbayasgalan commented on benchmarks/dynamo/pr_time_benchmarks/expected_results.csv:
```diff
@@ -38,7 +38,7 @@ update_hint_regression,compile_time_instruction_count,1719000000,0.1
 
 
-sum_floordiv_regression,compile_time_instruction_count,966100000,0.1
+sum_floordiv_regression,compile_time_instruction_count,1483069574,0.1
```
It actually decreased lol which i guess is expected?

avikchaudhuri replied:
Nope. :) And in the most recent update it's even higher.
This issue: https://github.com/pytorch/pytorch/issues/134133
Fixed here? https://github.com/pytorch/pytorch/pull/134150

Something to do with unbacked symints going wild...

tugsbayasgalan committed:
Update on "Migrating some more callsites"


Differential Revision: [D83009423](https://our.internmc.facebook.com/intern/diff/D83009423)

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx ipiszy chenyang78 kadeng muchulee8 amjames chauhang aakhundov coconutruben Lucaskabela

[ghstack-poisoned]

tugsbayasgalan committed:
Update on "Migrating some more callsites"


Differential Revision: [D83009423](https://our.internmc.facebook.com/intern/diff/D83009423)

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx ipiszy chenyang78 kadeng muchulee8 amjames chauhang aakhundov coconutruben Lucaskabela

[ghstack-poisoned]

tugsbayasgalan commented:
@tugsbayasgalan has imported this pull request.  If you are a Meta employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D83009423).

tugsbayasgalan committed:
Update on "Migrating some more callsites"


Differential Revision: [D83009423](https://our.internmc.facebook.com/intern/diff/D83009423)

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx ipiszy chenyang78 kadeng muchulee8 amjames chauhang aakhundov coconutruben Lucaskabela

[ghstack-poisoned]

tugsbayasgalan committed:
Update on "Migrating some more callsites"


Differential Revision: [D83009423](https://our.internmc.facebook.com/intern/diff/D83009423)

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx ipiszy chenyang78 kadeng muchulee8 amjames chauhang aakhundov coconutruben Lucaskabela

[ghstack-poisoned]

tugsbayasgalan commented:
@tugsbayasgalan has imported this pull request.  If you are a Meta employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D83009423).

tugsbayasgalan committed:
Update on "Migrating some more callsites"


Differential Revision: [D83009423](https://our.internmc.facebook.com/intern/diff/D83009423)

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx ipiszy chenyang78 kadeng muchulee8 amjames chauhang aakhundov coconutruben Lucaskabela

[ghstack-poisoned]

tugsbayasgalan committed:
Update on "Migrating some more callsites"


Differential Revision: [D83009423](https://our.internmc.facebook.com/intern/diff/D83009423)

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx ipiszy chenyang78 kadeng muchulee8 amjames chauhang aakhundov coconutruben Lucaskabela

[ghstack-poisoned]

tugsbayasgalan committed:
Update on "Migrating some more callsites"


Differential Revision: [D83009423](https://our.internmc.facebook.com/intern/diff/D83009423)

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx ipiszy chenyang78 kadeng muchulee8 amjames chauhang aakhundov coconutruben Lucaskabela

[ghstack-poisoned]
